Lib/test/support/__init__.py

```diff
@@ -1624,21 +1624,23 @@ def gc_collect():
     longer than expected.  This function tries its best to force all garbage
     objects to disappear.
     """
-    gc.collect()
-    if is_jython:
-        time.sleep(0.1)
-    gc.collect()
-    gc.collect()
+    # gc.collect()
+    # if is_jython:
+    #     time.sleep(0.1)
+    # gc.collect()
+    # gc.collect()
+    pass
 
 @contextlib.contextmanager
 def disable_gc():
-    have_gc = gc.isenabled()
-    gc.disable()
-    try:
-        yield
-    finally:
-        if have_gc:
-            gc.enable()
+    # have_gc = gc.isenabled()
+    # gc.disable()
+    # try:
+    #     yield
+    # finally:
+    #     if have_gc:
+    #         gc.enable()
```

Comment:
You should probably still yield here, since it's a context manager. Unless `@contextlib.contextmanager` works with non-generators.

Reply:
Done